lib/api/createReadStream.js

```diff
@@ -33,6 +34,7 @@ module.exports = function createReadStream(path, options, cb) {
     var close = request.bind(undefined, 'close', file, connection);
 
     var stream = new Readable();
+    stream.fileSize = fileSize;
```

Comment:
Is it standard?

Are there some widely used streams that used such a property?

Reply:
No it's not standard, we could add it as a third parameter in the callback as well if you prefer that? This was just an easy way for me to get the fileSize.

Reply:
I was just wondering about the name of the property but `fileSize` is good enough.